.env_sample

```diff
@@ -55,7 +55,7 @@ AWS_SECRET_ACCESS_KEY=testsecret
 AWS_STORAGE_BUCKET_NAME=public
 AWS_STORAGE_PRIVATE_BUCKET_NAME=private
 # NOTE! port 9000 here should match $MINIO_PORT
-AWS_S3_ENDPOINT_URL=http://docker.for.mac.localhost:9000/
+AWS_S3_ENDPOINT_URL=http://172.17.0.1:9000/
```

Comment:
Can we revert this and move this information to `README.md`? (already did this on the call, I believe)